dhall/features/network.dhall

```diff
@@ -0,0 +1,15 @@
+\(cluster : ../types/cluster.type) ->
+\(os: ../types/os.type) ->
+\(launcher : ../types/launcher.type) ->
+\(topology : ../types/topology.type) ->
+{ networkConfigurationYaml = {getConfigurationYamlPath = os.osConfigurationYaml}
+, networkTopology          = {getTopology = os.osNodeArgs.naTopology}
+, networkX509ToolPath      = {getX509ToolPath = os.osX509ToolPath}
+, networkTlsPath           = {getTlsPath = os.osNodeArgs.naTlsPath}
+, networkHost              = {getHost = cluster.ccfgRelays}
+, networkValency           = {getValency = topology.wallet.wtcfgValency}
```

Comment:
Yep, these ones might be a bit more difficult to sort out, I would say that they belong in the wallet.
In any case, no point in having duplicate parameters.